### DISCUSSION TITLE
Changing borg backup time?

### DISCUSSION THREAD

**Discussion by emelbardis** [nextcloud/all-in-one#2860] [Questions] [answered]
I set a time when installing AIO but see that the option is now missing after I set it up.
How can I change the time backups are triggered?

Thanks

**Comment by szaimen** [comment]
Hi, can you post a screemshot of what you are currently seeing in the aio interface?

**Comment by emelbardis** [comment]
Sure. I changed it manually and hope the syntax is correct. Here is the expanded borg view from my interface
 
![23-06-25 16-35-40 0137](https://github.com/nextcloud/all-in-one/assets/20196908/2b228e6f-5556-4b2e-8deb-4a125365fd02)
![Screenshot from 2023-06-25 16-33-50](https://github.com/nextcloud/all-in-one/assets/20196908/676848b0-2db9-4a04-b5c2-5140ca4522ae)

**Comment by szaimen** [comment, marked as answer]
You can adjust the time by first disabling daily backups and then re-enabling them with an adjusted time. Is this not clear?
![image](https://github.com/nextcloud/all-in-one/assets/42591237/eff04c5b-b952-4f74-bf1a-06ceb2d5fc50)

**Reply by stefan-niedermann** [reply]
Found this answer after searching for the same issue: I can confirm that this is unclear from a users perspective. Maybe change the wording of the button to <kbd>Change or disable daily backups</kbd>. Alternatively place a second button next to the <kbd>Disable daily backups</kbd> with the label <kbd>Change backup settings</kbd> or so :wink:

**Reply by szaimen** [reply]
Good idea! Will be adjusted with https://github.com/nextcloud/all-in-one/pull/4399

**Reply by szaimen** [reply]
This is now released with v8.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by emelbardis** [comment]
Well, I hadn't tried disabling to see if I then saw an option to reenable with a new time so it wasn't clear to me.
Perhaps a sentence stating: "To change your backup time, first disable Daily Backups and then re-enable them with your new backup time" or something

**Reply by szaimen** [reply]
Sounds good!

**Reply by Croydon** [reply]
Thank you for the change!

I wasn't aware that this time can be changed either since the was no indication previously.

**Comment by szaimen** [comment]
Addressed by https://github.com/nextcloud/all-in-one/pull/2873
